dali/operators/reader/loader/video/frames_decoder_gpu.h

```diff
@@ -66,8 +66,12 @@ class DLL_PUBLIC FramesDecoderGpu : public FramesDecoder {
 
   int NextFramePts() { return index_[NextFrameIdx()].pts; }
 
+  void SetCudaStream(cudaStream_t stream) { stream_ = stream; }
```

Comment:
Do you use it anywhere now? Or we keep going on the default stream?

Reply:
No, removed this. Stream is set during the construction now.